examples/distroless/Dockerfile

```diff
@@ -0,0 +1,8 @@
+# ch-test-scope: standard
+# Distroless is a Google project providing slim images that contain runtime
+# dependencies only. https://github.com/GoogleContainerTools/distroless
+FROM gcr.io/distroless/python3
```

Comment:
Of the distroless tags what reason, if any, do we pick the python3 one? Perhaps comment on our choice here.

Reply:
I chose the`python3` image because of all of their base images it is the simplest to test. I will add a comment to this effect.